### ISSUE TITLE
locomotive.gemspec and Gemfile need to be kept in sync

### ISSUE THREAD

**Issue by mscottford** [locomotivecms/engine#219]
TL;DR I'm willing to make this change, but I thought it would be best to talk about why it should be made.

`locomotive.gemspec` and `Gemfile` have gotten out of sync with each other a couple of times over the last couple of months. I'd like to recommend that `Gemfile` be redefined to reference `locomotive.gemspec` using the `gemspec` command. All dependencies (including development dependencies) will then be defined in `locomotive.gemspec`. This should prevent future issues like the one I fixed in pull request #218. 

References: 
- http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/
- http://yehudakatz.com/2010/04/02/using-gemspecs-as-intended/
- (See bottom of the "Setup" section) http://coding.smashingmagazine.com/2011/06/23/a-guide-to-starting-your-own-rails-engine-gem/
